rsciio/_hierarchical.py

```diff
@@ -239,7 +239,11 @@ def _read_array(group, dataset_key):
             key = "ragged_shapes"
         if key in group:
             ragged_shape = group[key]
+            ragged_shape = np.array(
+                ragged_shape
+            )  # cast to numpy array to stop multiple reads
```

Comment:
It is not obvious what is meant with the comment (at least to me!) Can you elaborate a bit more please?

Reply:
Done!